website/site/content/docs/collection-types.md

```diff
@@ -17,20 +17,21 @@ Folder collections represent one or more files with the same format, fields, and
 
 Unlike file collections, folder collections have the option to allow editors to create new items in the collection. This is set by the boolean `create` field.
 
-**Note:** Folder collections must have at least one field with the name "title" for creating new entry slugs. That field should use the default "string" widget. The "label" for the field can be any string value.
+**Note:** Folder collections must have at least one field with the name `title` for creating new entry slugs. That field should use the default `string` widget. The `label` for the field can be any string value. See the [collections](https://www.netlifycms.org/docs/configuration-options/#collections) section for details on how collections and fields are configured. 
```

Comment:
I think I would call this link the "Collections reference doc" here, as well. For me, "collections section" implies that it's another section of the same page.